ironfish/src/rpc/clients/client.ts

```diff
@@ -803,4 +811,32 @@ export abstract class RpcClient {
       ).waitForEnd()
     },
   }
```

Comment:
We were missing this previously

Reply:
I thought this was intentional because we didn't want to expose this in the sdk just yet.

Reply:
It is already exposed via the routes, we just don't have typed calls for the RPC.